Kitodo-DataEditor/src/main/java/org/kitodo/dataeditor/ruleset/xml/Reimport.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.dataeditor.ruleset.xml;
+
+import javax.xml.bind.annotation.XmlEnum;
+import javax.xml.bind.annotation.XmlEnumValue;
+
+import org.apache.commons.lang3.ArrayUtils;
+
+/**
+ * This class is a backing bean for the XML attribute reimport in the ruleset.
+ * With it, JAXB can map the attribute to an enum.
+ */
+@XmlEnum(String.class)
```

Comment:
```suggestion
@XmlEnum()
```
Redundant default parameter assignenment